libs/template/themes/design-tokens/src/common-tokens.ts

```diff
@@ -1,6 +1,21 @@
 import { ThemeToken } from '@spryker-oryx/experience';
 
 export const commonTokens: ThemeToken = {
+  /** @deprecated use oryx-shadow instead */
+  elevation: {
```

Comment:
BC?

Reply:
And why deprecated?

Reply:
no BC until it's removed. We moved to `shadow` prefix rather than `elevation`.